tests/unit/date-mixin.spec.ts

```diff
@@ -81,6 +81,7 @@ describe('Date Mixin', () => {
   xit('returns correct values for yyyyMmDdToApi()', () => {
     expect(vm.yyyyMmDdToApi('2021-01-01')).toBe('2021-01-01T08:00:00+00:00') // PST
     expect(vm.yyyyMmDdToApi('2021-07-01')).toBe('2021-07-01T07:00:00+00:00') // PDT
+    expect(vm.yyyyMmDdToApi('2023/05/03')).toBe('2023-05-03T07:00:00+00:00')
```

Comment:
Curious here: you added a test case for the change you did (which is great) but the test itself is excluded. Do we keep the excluded tests as is and not include in the test suite?

Reply:
I ran the unit tests locally:

![image](https://user-images.githubusercontent.com/10002889/236002169-38cf1fbe-a7f6-4741-bb4e-dae16febbbee.png)

This unit test previously failed in the GH check (due to a different Node version or something). I don't know if it would still fail in GH but that's more than I want to get into right now.

However, I think this unit test is still valuable to have for the future when we can un-exclude it. (I think of this test as being "blocked", not "useless".) Also I wouldn't want to just delete the test because having it shows we thought about that the method under test does.

Reply:
Totally makes sense to not delete. Good reference for me while adding unit tests in future :)